completion/waffle.py

```diff
@@ -4,22 +4,10 @@
 """
 
 
-from edx_toggles.toggles import WaffleSwitch, WaffleSwitchNamespace
+from edx_toggles.toggles.__future__ import WaffleSwitch
 
-# Namespace
+# The switch and namespace names variables are preserved for backward compatibility
 WAFFLE_NAMESPACE = "completion"
-
-
-def waffle():
```

Comment:
Another option is leaving this, setting a `deprecated` custom attribute, and making this v3.2.6 instead of v4.0.0 with no backward compatible changes.  That way, if we missed anything where this was used outside of tests, we could catch it before its removal.

Reply:
If this breaks anything then the fix should be relatively straightforward. edx-platform should not be impacted, so we should only be concerned by other applications. I think this is a reasonable risk.